_includes/dc/setup.html

```diff
@@ -40,7 +40,10 @@
 {% endcomment %}
 
 {% elsif site.curriculum == "dc-genomics" %}
-<p>The setup instructions for the Data Carpentry Genomics workshops can be found at <a href="https://datacarpentry.org/genomics-workshop/index.html#setup">the workshop overview site</a></p>.
+<p>The setup instructions for the Data Carpentry Genomics workshops can be found at <a href="https://datacarpentry.org/genomics-workshop/index.html#setup">the workshop overview site</a>.
+As the workshop will be run locally, the software listed in <a href="https://datacarpentry.org/genomics-workshop/index.html#option-b-using-the-lessons-on-your-local-machine">"Option B"</a> has to be installed by all participants.</p>
```

Comment:
```suggestion
In this workshop, we will run commands and programs on local machines, hence falling within <a href="https://datacarpentry.org/genomics-workshop/index.html#option-b-using-the-lessons-on-your-local-machine">"Option B"</a> of the setup instructions.</p>
```
Hopefully more beginner-friendly wording? Also, I thought it would be confusing to read "has to be installed by all participants" and, one line later, "please wait before installing ..." Anyway, this will be updated shortly again!